### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by luandy64**
# Description of change
This PR gets Circle passing again

# Manual QA steps
 - 
 
# Risks
 - Low
 
# Rollback steps
 - revert this branch

### CODE REVIEW

**Review by luandy64 — COMMENTED** [review]
Here is a review comment

**Comment by luandy64** [comment]
Here is a PR comment?

**Review by andylu-test — COMMENTED** [review]
Commenting in the review

**Comment by andylu-test** [comment]
Commenting on the PR

**Review by cosimon — APPROVED** [review]

File: tests/test_github_start_date.py [hunk 1]
```diff
@@ -18,7 +18,7 @@ def test_run(self):
         """Instantiate start date according to the desired data set and run the test"""
 
         self.start_date_1 = '2020-04-01T00:00:00Z'
-        self.start_date_2 = self.timedelta_formatted(self.start_date_1, days=365)
+        self.start_date_2 = '2021-06-10T00:00:00Z'
```
Comment (luandy64) [inline tests/test_github_start_date.py:21]: Comment in the code, in a review

**Review by luandy64 — COMMENTED** [review]
Comment with a comment in the review

**Pull request merged**